apps/dashboard/app/models/script.rb

```diff
@@ -137,10 +140,10 @@ def save
     false
   end
 
-  def destroy
-    FileUtils.remove_file(script_file)
-    FileUtils.remove_file(job_log_file)
-    FileUtils.remove_file(cache_file_path) if cache_file_exists?
+  def destroy!
```

Comment:
The rails convention here is that `!` methods raise errors where ones without the exclamation point safely capture errors and return true or false.

Seems like this method is the latter (method with no `!`)?

Reply:
Sorry, yes. This is the second time I do the same, I got confused with side effects.
Sorry, updating....

Reply:
fixed